PRODUCTIONISATION.md

```diff
@@ -19,26 +19,26 @@
 
 Ensuring that your project adheres to the following RFCs is a good first step as these will be referenced further below:
 
+* [RFC 008 - Server naming convention][rfc_008]
 * [RFC 011 - Editorconfig][rfc_011]
 * [RFC 012 - Makefile standards][rfc_012]
 * [RFC 013 - Code repository standards][rfc_013]
 
 ### 1. General requirements
 
 * [ ] 1.1 Application source code hosted with a Git VCS service
-* 1.2 No linting violations
-  * [ ] 1.2.1 No style related linting violations and no overrides permitted
```

Comment:
> no overrides permitted
not sure about this. In practice, to be honest, sometimes we disable Rubocop because he was too restrictive (maybe we had to configure it in better way)

Reply:
Was this for specific cops? I think for style related cops, I've not seen a reason to disable them, but certainly for code complexity related cops, such as method lengths, ABC size etc, sometimes it is best to disable them on a method (or class basis) as there is no cleaner way of writing the code that doesn't lose good readability.

Reply:
I think it is not just about `rubocob`.
Sometimes I used to disable Rubocop for a certain metric only within a block of code.
For example:

```
# rubocop:disable Metrics/LineLength
...
# rubocop:enable Metrics/LineLength
```

The reasons were eighter:
- missing time to write better code (this might be considered tech debt to be fixed)
- or Rubocob suggested a really weird code style

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/25720ea6b439d9ec94512ab3ff2cd7b0819d5f9b.